### PR TITLE
Space Travelers Hub 

### PR DESCRIPTION
**Pull request by Chimoneg27**
For this project me and my partner did the following

- Use React components
- Use React props
- Use React Router
- Connected with the SpaceX API to fetch data for the rockets and missions page
- Created the my profile page to tracked reserved rockets and active missions 
- Connect React and Redux
- Handle events in a React app
- Write unit tests with React Testing Library
- Use styles in a React app
- Use React hooks
- Apply React best practices and language style guides in code
- Use store, actions and reducers in React
- Perform a code review for a team member
- Removed linter errors
- Added a live demo and video walkthrough in our README file
- Used gitflow
- Made required changes 

### Note

We added other tests. As such some of the tests connect to other components

### CODE REVIEW

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/test/Mission.test.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import '@testing-library/jest-dom/extend-expect';
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import configureMockStore from 'redux-mock-store';
+import Missions from '../components/Missions';
+
+const mockStore = configureMockStore([]);
+
+test('renders missions component correctly', async () => {
```
[the remark this answers is not in the document]
Comment (Chimoneg27) [reply src/test/Mission.test.js:10]: How many components mut we test?

[the remark this answers is not in the document]
Comment (gemmen29) [reply src/test/Mission.test.js:10]: Hello @Chimoneg27, You need to add tests for all your components and functions as per the requirement of this project.
image

**Review by KDlamini — APPROVED** [review]
### Hi team @Chimoneg27, @AlselwiAisha  🙋‍♀️,  

Excellent work! 💯 
You have done an outstanding job implementing all the required changes for thje React Space X Traveler's Hub. 🎊 🥳 
Thank you for showing committment and dedication in submitting a profound project 👏🏾 🚀 
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!

## HIGHLIGHTS 🌟 

✔️ Great PR title and description
✔️ Events, fetch, and good state management with Redux
✔️ Good UI render and navigation for all pages
✔️ Well written tests for components and actions
✔️ Well documented and professional README
✔️ Good commit history and meaningful commit messages

# Status: Approved ✔️ 

oscars-standing-ovation
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**